package.json

```diff
@@ -24,31 +24,31 @@
     "@types/node": "^14.14.7",
     "@types/source-map-support": "^0.5.2",
     "@types/yup": "^0.28.3",
-    "@typescript-eslint/eslint-plugin": "^4.7.0",
-    "@typescript-eslint/parser": "^4.7.0",
+    "@typescript-eslint/eslint-plugin": "^4.11.0",
+    "@typescript-eslint/parser": "^4.11.0",
     "babel-eslint": "10.x",
     "eslint": "^7.13.0",
-    "eslint-config-prettier": "^6.15.0",
+    "eslint-config-prettier": "^7.1.0",
     "eslint-config-react-app": "^6.0.0",
     "eslint-plugin-flowtype": "5.x",
     "eslint-plugin-import": "2.x",
     "eslint-plugin-jsx-a11y": "6.x",
-    "eslint-plugin-prettier": "^3.1.4",
-    "eslint-plugin-react": "7.x",
+    "eslint-plugin-prettier": "^3.3.0",
+    "eslint-plugin-react": "^7.22.0",
     "eslint-plugin-react-hooks": "4.x",
     "husky": "^4.3.0",
     "istanbul-reports": "^3.0.2",
-    "lint-staged": "^10.4.0",
-    "markdownlint-cli": "^0.24.0",
+    "lint-staged": "^10.5.3",
+    "markdownlint-cli": "^0.26.0",
     "pkg": "^4.4.9",
-    "prettier": "^2.1.2",
+    "prettier": "^2.2.1",
     "redux-devtools-extension": "^2.13.8",
     "rimraf": "^3.0.0",
     "source-map-support": "^0.5.19",
-    "ts-loader": "^8.0.11",
-    "ts-node": "^9.0.0",
-    "typescript": "4.0.5",
-    "wsrun": "^5.2.0"
+    "ts-loader": "^8.0.13",
+    "ts-node": "^9.1.1",
+    "typescript": "4.1.3",
```

Comment:
had some fragmentation here, parts of the code base already asked for this. should aim for a common typescript version across code base.